test/onigumo_cli_test.exs

```diff
@@ -33,15 +34,18 @@ defmodule OnigumoCLITest do
     end
 
     test("run CLI with no arguments") do
-      assert_raise(MatchError, fn -> Onigumo.CLI.main([]) end)
+      output = capture_io(fn -> Onigumo.CLI.main([]) end)
+      assert String.starts_with?(output, "Usage: ./onigumo ")
     end
 
     test("run CLI with more than one argument") do
-      assert_raise(MatchError, fn -> Onigumo.CLI.main(["Downloader", "Parser"]) end)
+      output = capture_io(fn -> Onigumo.CLI.main(["Downloader", "Parser"]) end)
+      assert String.starts_with?(output, "Usage: ./onigumo ")
     end
 
     test("run CLI with invalid switch") do
-      assert_raise(OptionParser.ParseError, fn -> Onigumo.CLI.main(["--help"]) end)
+      output = capture_io(fn -> Onigumo.CLI.main(["--help"]) end)
+      assert String.starts_with?(output, "Usage: ./onigumo ")
```

Comment:
The _capture_io_ and _start_with?_ logic is quite complex and is repeated three times here. Could it be refactored to a private function acctepting just the _fn_?